deploy/docs/V1_Migration_doc.md

```diff
@@ -0,0 +1,156 @@
+| sumologic.logFormat 	| fluentd.logs.output.logFormat 	|
+| sumologic.flushInterval 	| fluentd.buffer.flushInterval 	|
+| sumologic.numThreads 	| fluentd.buffer.numThreads 	|
+| sumologic.chunkLimitSize 	| fluentd.buffer.chunkLimitSize 	|
+| sumologic.queueChunkLimitSize 	| fluentd.buffer.queueChunkLimitSize 	|
+| sumologic.totalLimitSize 	| fluentd.buffer.totalLimitSize 	|
+| sumologic.sourceName 	| fluentd.logs.containers.sourceName 	|
+| sumologic.sourceCategory 	| fluentd.logs.containers.sourceCategory 	|
+| sumologic.sourceCategoryPrefix 	| fluentd.logs.containers.sourceCategoryPrefix 	|
+| sumologic.sourceCategoryReplaceDash 	| fluentd.logs.containers.sourceCategoryReplaceDash 	|
+| sumologic.addTimestamp 	| fluentd.logs.output.addTimestamp 	|
+| sumologic.timestampKey 	| fluentd.logs.output.timestampKey 	|
+| sumologic.verifySsl 	| fluentd.verifySsl 	|
+| sumologic.excludeContainerRegex 	| fluentd.logs.containers.excludeContainerRegex 	|
+| sumologic.excludeHostRegex 	| fluentd.logs.containers.excludeHostRegex 	|
+| sumologic.excludeNamespaceRegex 	| fluentd.logs.containers.excludeNamespaceRegex 	|
+| sumologic.excludePodRegex 	| fluentd.logs.containers.excludePodRegex 	|
+| sumologic.fluentdLogLevel 	| fluentd.logLevel 	|
+| sumologic.watchResourceEventsOverrides 	| fluentd.events.watchResourceEventsOverrides 	|
+| sumologic.fluentd.buffer 	| fluentd.buffer.type 	|
+| sumologic.fluentd.autoscaling.* 	| fluentd.autoscaling.* 	|
+| sumologic.k8sMetadataFilter.watch 	| fluentd.logs.containers.k8sMetadataFilter.watch 	|
+| sumologic.k8sMetadataFilter.verifySsl 	| fluentd.logs.containers.k8sMetadataFilter.verifySsl 	|
+| sumologic.k8sMetadataFilter.cacheSize 	| fluentd.metadata.cacheSize 	|
+| sumologic.k8sMetadataFilter.cacheTtl 	| fluentd.metadata.cacheTtl 	|
+| sumologic.k8sMetadataFilter.cacheRefresh 	| fluentd.metadata.cacheRefresh 	|
+| deployment.* 	| fluentd.statefulset.* 	|
+| eventsDeployment.* 	| fluentd.eventsStatefulset.* 	|
+
+
+   - `sumologic.kubernetesMeta` and `sumologic.kubernetesMetaReduce` have been removed. The default log format (`fluentd.logs.output.logFormat`) is `fields`, which removes the relevant metadata from the JSON body of the logs, making these configs no longer necessary.
+   - `sumologic.addStream` and `sumologic.addTime` (default values were `true`) have been removed; the default behavior will remain the same. To preserve the behavior of `addStream = false` or `addTime = false`, you can add the following config to the `values.yaml` file:
+
+```yaml
+fluentd:
+  logs:
+    containers:
+      extraFilterPluginConf:
+        <filter **>
+          @type record_modifier
+          remove_keys stream, time
+        </filter>
+```
+
+Until now, Helm users have not been able to modify their Fluentd configuration outside of the specific parameters that we exposed in the `values.yaml` file. Now, we expose the ability to modify the Fluentd configuration as needed. 
```

Comment:
We need a document with some examples of these and should link back to this.  I think we should show some simple examples that illustrate how to use the new configuration options as well as the list of all properties in the values.yaml that were added to enable this.

Reply:
will create a new document with examples.